com.unity.multiplayer.mlapi/Tests/Runtime/com.unity.multiplayer.mlapi.runtimetests.asmdef

```diff
@@ -1,11 +1,23 @@
 {
     "name": "Unity.Multiplayer.MLAPI.RuntimeTests",
+    "rootNamespace": "",
     "references": [
-        "Unity.Multiplayer.MLAPI.Runtime"
-    ],
-    "optionalUnityReferences": [
-        "TestAssemblies"
+        "Unity.Multiplayer.MLAPI.Runtime",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner",
+        "Unity.Multiplayer.MLAPI.Editor"
     ],
     "includePlatforms": [],
-    "excludePlatforms": []
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": true,
+    "precompiledReferences": [
+        "nunit.framework.dll"
+    ],
+    "autoReferenced": false,
+    "defineConstraints": [
+        "UNITY_INCLUDE_TESTS"
+    ],
+    "versionDefines": [],
+    "noEngineReferences": false
```

Comment:
not all of these changes are needed.

this would be enough:
```diff
+        "Unity.Multiplayer.MLAPI.Editor"
```

Reply:
That's the only change I did. Unity generated the rest.

Reply:
no worries, I'll take care of minor things.